python/ntp/commands/create/tarball/__init__.py

```diff
@@ -70,7 +70,9 @@ def __prepare_cmssw_src(self):
         # Note that directories are only looked-for and added under the src/ folder.
         # /data/ subdirs contain data files needed by the code
         # /interface/ subdirs contain C++ header files needed e.g. by ROOT6
-        directories = ['data', 'interface', 'python']
+        directories = [os.path.join(CMSSW_SRC, 'BristolAnalysis', 'NTupleTools','data'), 
+            os.path.join(CMSSW_SRC, 'BristolAnalysis', 'NTupleTools','interface'),
```

Comment:
Not what I meant. These paths are already saved in `cmssw_tar`!
What I meant is that in your scripts, please point to the `os.path.join(CMSSW_SRC, 'BristolAnalysis', 'NTupleTools','data'` path. No changes in `create tarball` required!